test-suite/output/SearchPattern.out

```diff
@@ -13,35 +13,39 @@ Nat.testbit: nat -> nat -> bool
 Nat.eqb: nat -> nat -> bool
 (use "About" for full details on implicit arguments)
 Nat.two: nat
-Nat.one: nat
 Nat.zero: nat
+Nat.one: nat
```

Comment:
Can we fix `SearchPattern` to sort its results or something so that there's not so much churn?  (cc @herbelin ?)

Reply:
Do you have a sorting order in mind?

Reply:
Alphanumeric?  Really, any sorting order that is stable under insertion is fine (that is, inserting things should not reorder the things already in the map), and ideally a fast sorting method, if sorting is a bottleneck.

Reply:
Actually, there is already an ordering, based on a heuristics computing size and number of distinct symbols. It would be easy to tweak it by modifyng line 161 of `vernac/search.ml`  to something like:
```
 let compare (gref1,_,_,p1) (gref2,_,_,p2) =
    let n = pervasives_compare p1 p2 in
    if n = 0 then some_relevant_order gref1 gref2 else n
```
But I guess, you would need first to turn the full references into qualified identifiers before sorting them, and there is order ready on qualified identifiers. So, maybe for a next time...